### PR TITLE
Add Gedling Borough Council (unofficial)

### PR DESCRIPTION
**Pull request by jamesmacwhite**
Hello! I saw #1806 recently and looked at converting the existing bin collection schedules provided by Gedling Borough Council. I created a repository to host all the iCal files: https://github.com/jamesmacwhite/gedling-borough-council-bin-calendars. 

## General bin schedules

There's a total of 20 bin collection schedules depending on the area within the borough you are located in.

## Garden waste collection schedules

There's a total of 10 labelled A - J for each weekday.

## About the iCal files

As these iCal files are generated by an unofficial source, I don't know if there's any specific handling needed regarding contributing or making it clear these are not directly provided by Gedling Borough Council. I have marked the title with (unofficial) my repository where the iCal files are also states this on a disclaimer.

Please let me know if there's anything I should be doing due to the unofficial nature scenario.

Tested one of the .ics calendars (Wednesday G2) in Home Assistant for reference:

image

This calendar data should align to the official printed calendar provided by Gedling Borough Council:

https://apps.gedling.gov.uk/refuse/data/WednesdayG2-2024.pdf

Tested configuration.yaml:

```yaml
sensor:
  - platform: waste_collection_schedule
    name: Gedling Borough Council bin collection

waste_collection_schedule:
  sources:
    - name: ics
      args:
        url: https://raw.githubusercontent.com/jamesmacwhite/gedling-borough-council-bin-calendars/main/ical/gedling_borough_council_wednesday_g2_bin_schedule.ics
```

### CODE REVIEW

File: doc/ics/yaml/gedling_gov_uk.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title: Gedling Borough Council (unofficial)
+url: https://github.com/jamesmacwhite/gedling-borough-council-bin-calendars
+howto: |
+   - Gedling Borough Council does not provide bin collections in the iCal calendar format directly.
+   - The iCal calendar files have been generated from the official printed calendars and hosted on GitHub for use.
+   - Go to: https://apps.gedling.gov.uk/refuse/search.aspx and enter your street name to find your bin day/garden waste collection schedule.
+   - Find the matching .ics file from within the ical folder from the unofficial repository.
```
Comment (5ila5) [inline doc/ics/yaml/gedling_gov_uk.yaml:7]: People not familiar with GitHub often struggle to download files from GitHub: I'd add a not that you have to click on the file and copy the link by clicking `copy raw file` to then use this as URL parameter

Comment (jamesmacwhite) [reply doc/ics/yaml/gedling_gov_uk.yaml]: Thanks. I've added further guidance/information.

**Comment by 5ila5** [comment]
Thanks for your contribution

**Comment by jamesmacwhite** [comment]
Thanks for approving it. I hope it is useful to other resident in Gedling.

**Pull request merged**